package.json

```diff
@@ -14,6 +14,8 @@
     "lint:fix": "yarn lint --fix",
     "setup": "tsc -b ./packages/tsconfig.json ",
     "test": "jest",
+    "test:unit": "jest unit",
+    "test:integration": "jest synth-index",
```

Comment:
In the future this will become just:
```
"test:integration": "jest integration",
```
?

Reply:
yeah